### PR TITLE
Caplin: Fixed races and Beacon API inconsenstincies

### PR DESCRIPTION
**Pull request by Giulio2002**

### CODE REVIEW

File: cl/sentinel/handlers/light_client.go [hunk 1]
```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/libp2p/go-libp2p/core/network"
 )
 
+const maxLightClientsPerRequest = 100
```
Comment (bayram98) [inline cl/sentinel/handlers/light_client.go:12]: @Giulio2002 How about changing the value of `c.beaconConfig.MaxRequestLightClientUpdates` from config? 

https://github.com/ledgerwatch/erigon/blob/d6a10a4aedf772f49af8789aeaa4a7c8f8f2ef0e/cl/clparams/config.go#L809

Comment (Giulio2002) [reply cl/sentinel/handlers/light_client.go:12]: i dont think that should actually be kept in beacon config. i think this is fine. i removed that field btw from the config

Comment (bayram98) [reply cl/sentinel/handlers/light_client.go:12]: Okay. Then it is good.

**Review by bayram98 — APPROVED** [review]
@Giulio2002 LGTM!

There are Lint check fails on [cl/fork/fork_test.go](https://github.com/ledgerwatch/erigon/pull/9610/files#diff-48e1d1d13b9223a4637cd67f322522ac8fc024a9c8eacdd131b56b907501fc84)
Can you check it before merge the PR?

**Pull request merged**